Packs/CoreAlertFields/IncidentFields/incidentfield-Event_Type.json

```diff
@@ -28,5 +28,39 @@
     "useAsKpi": false,
     "version": -1,
     "fromVersion": "6.5.0",
-    "x2_fields": "event_type"
+    "x2_fields": "event_type",
+    "Aliases": [
+        {
+          "cliName": "qualysfimeventtype",
+          "type": "shortText"
+        },
+        {
+          "cliName": "arguseventtype",
+          "type": "shortText"
+        },
+        {
+          "cliName": "eventtype",
+          "type": "shortText"
+        },
+        {
+          "cliName": "googlecloudsccfindingsourcepropertieseventtype",
+          "type": "shortText"
+        },
+        {
+          "cliName": "laceworkeventtype",
+          "type": "shortText"
+        },
+        {
+          "cliName": "qualysfimeventtype",
+          "type": "shortText"
+        },
+        {
+          "cliName": "solarwindseventtype",
+          "type": "shortText"
+        },
+        {
+          "cliName": "taniumtreatresponsescanconfigid",
```

Comment:
Is this related?

Reply:
![image](https://user-images.githubusercontent.com/88268646/152372601-86e691be-31ec-45e1-8644-5bc4194fc668.png)
Followed this record. Seems like I should take the one from above right? @David-BMS

Reply:
Yep

Reply:
Done